archivers/libzzip/Portfile

```diff
@@ -64,14 +57,43 @@ post-destroot {
 }
 
 variant sdl description {Enable SDL support} {
-    depends_lib-append      port:libsdl
+    depends_lib-append      port:libsdl2
 
-    configure.args-append   --enable-sdl
+    configure.args-append   -DZZIPSDL=ON
 
     post-destroot {
         xinstall -m 0644 ${worksrcpath}/docs/README.SDL ${destroot}${docdir}
     }
 }
 
-test.run        yes
-test.target     check
+variant python35 conflicts python36 python37 python38 python39 description {Build using Python 3.5} {
+    depends_lib-append \
+        port:python35
```

Comment:
Before your changes, python27 was a build dependency. Is it now really needed at runtime as well?

Reply:
Nope, my bad.